src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -731,4 +756,18 @@ class GroupReadsByUmi
         r1[String](this.rawTag)
     }
   }
+
+  /** Sets the duplicate flags on all reads within all templates.  */
+  private def setDuplicateFlags(group: Seq[Template]): Unit = {
+    val nonDuplicateTemplate = group.minBy { template =>
```

Comment:
Do we want `maxBy` here?  To my eye, htsjdk's duplicate score returns higher values as better:

https://github.com/samtools/htsjdk/blob/38dbd0e6b87a2bd5791bb81399f2b5ec1008773e/src/main/java/htsjdk/samtools/DuplicateScoringStrategy.java#L73-L116

For example, if the duplicate score is the sum of base qualities, then don't we want the template with the highest sum of base qualities?

Reply:
Good catch!